pkg/querier/queryrange/results_cache.go

```diff
@@ -301,7 +301,8 @@ func (s resultsCache) handleHit(ctx context.Context, r Request, extents []Extent
 	log, ctx := spanlogger.New(ctx, "handleHit")
 	defer log.Finish()
 
-	requests, responses, err := partition(r, extents, s.extractor)
+	const minCacheExtent = 5 * 60 * 1000 // 5 minutes
```

Comment:
Does it make any sense to make this configurable? (I leaning toward "no")

Reply:
I would use `(5 *time.Minute).Milliseconds()` to make it more readable.

Reply:
> Does it make any sense to make this configurable?

I would start hardcoded and eventually make it configurable if a good use case arise for that.